Modix.Bot/Modules/EmojiStatsModule.cs

```diff
@@ -62,6 +63,48 @@ public async Task BottomGuildOnlyEmojiStatsAsync()
             await ReplyAsync(embed: embed.Build());
         }
 
+        [Command()]
+        [Priority(-10)]
+        [Summary("Gets the usage stats for a specific user.")]
+        public async Task UserEmojiStatsAsync(
+            [Summary("The user to retrieve stats for.")]
+                IUser user)
+        {
+            var userId = user.Id;
+            var guildId = Context.Guild.Id;
+
+            var emojiStats = await _emojiRepository.GetEmojiStatsAsync(guildId, SortDirection.Ascending, 10, userId: userId);
+            var userTotalUses = await _emojiRepository.SearchSummariesAsync(new EmojiSearchCriteria()
```

Comment:
Is there a particular reason we're aggregating over a user's entire dataset in RAM, rather than within the database?